### DISCUSSION TITLE
Configuration instructions for Linux

### DISCUSSION THREAD

**Discussion by Yora0** [func-godot/func_godot_plugin#48] [Ideas]
The [configuration instructions in the manual](https://func-godot.github.io/func_godot_docs/FuncGodot%20Manual/FuncGodot%20Manual.html) are put in a way that is very unclear for Linux systems.

I am not able to identify my "map editor's game configuration folder or installation folder", "the location of your game's TrenchBroom Game Config folder", or " what your Map Editor considers the location of your project".

A dedicated configuration guide for setting up func-godot would be hugely beneficial for new users.

**Comment by RhapsodyInGeek** [comment]
If you are unable to identify those, you need to read your map editor's documentation and learn more about how your map editor works. Even on Linux systems you have locations for all of those as determined by your chosen map editor. Due to its complexity this plugin expects that you have at least an intermediate level of knowledge with your tools, including where they save all of their data and configurations. It is unfortunately not going to be aimed at beginners, at least not by the FuncGodot Team due to our limited time and resources.

**Comment by RisingThumb** [comment]
For knowing what is meant by "game configuration", it may help to read what the Trenchbroom documentation says on it https://trenchbroom.github.io/manual/latest/#game_configuration

> A dedicated configuration guide for setting up func-godot would be hugely beneficial for new users.

This already exists, and it's under the "Project Configuration" tab of the func_godot documentation. You will have to do a little bit of legwork yourself to figure out the appropriate file paths for things, but that's to be expected. As someone on Linux, I had no issues setting things up. The documentation is extensive and detailed, so read through it carefully
